pages/content/amp-dev/page-experience.html

```diff
@@ -1,8 +1,9 @@
 ---
-$title: Page Experience
+$title: AMP Page Experience tool
 $view: /views/custom.j2
-description: The new page experience tool.
+description: Optimize your AMP pages for Google's Page Experience ranking signal using AMP's page experience tool.
```

Comment:
Suggestion: "Analyze and optimize your AMP Pages for Google's Page Experience ranking signal".

Reply:
Done